examples/dato/README.md

```diff
@@ -0,0 +1,59 @@
+# Makeswift Dato Integration
+
+(Link to landing page):
+https://makeswift-examples-basic-typescript.vercel.app/
+
+This guide assumes that you're familiar with Makeswift and setting up a custom host. Read here to learn more: https://docs.makeswift.com/developer/app-router/installation
+
+## Using this example
+
+To quickly try this example either [deploy to Vercel](#deploy-this-example-to-vercel) or [use our CLI](#use-this-example-locally-with-the-makeswift-cli).
+
+In addition to your Makeswift API key, add your Dato API key in either `.env.local` or your environment variables in vercel.
+
+`NEXT_PUBLIC_DATO_CMS_API_TOKEN=your_token`
+
+---
+
+### Deploy this example to Vercel
+
+The deploy link below includes integrations with Dato and Makeswift.
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmakeswift%2Fmakeswift%2Ftree%2Fmain%2Fexamples%2Fdato&project-name=dato-makeswift-example&repository-name=dato-makeswift-example&redirect-url=https%3A%2F%2Fapp.makeswift.com&integration-ids=oac_51ryd7Pob5ZsyTFzNzVvpsGq,oac_nsrwzogJLEFglVwt2060kB0y&external-id=dato-makeswift)
+
+### Use this example locally with the Makeswift CLI
+
+Set up this example of Makeswift by running
+
+```bash
+npx makeswift@latest init --example=dato
+```
+
+## Local Development
+
+Run `pnpm codegen-ts` after copying the following Dato schema
+
+![alt text](https://github.com/user-attachments/assets/8f235b4b-ed0f-4b99-a018-15ceaa56f1ac)
+
+Note that the Structured Text type ("body") has a validation field under "Specify the blocks allowed" and "Image" in the combobox.
+
+We recommend following the same schema because the `.graphql` files provided assume the same schema.
```

Comment:
We could probably make this a bit stronger of a recommendation. Instead of "we recommend", we could be clear that our graphql queries are dependent on the same schema. Otherwise, it will break.